co2eq/car.js

```diff
@@ -0,0 +1,175 @@
+import {
+  ACTIVITY_TYPE_TRANSPORTATION,
+  TRANSPORTATION_MODE_CAR,
+} from '../definitions';
+
+export const modelName = 'car';
+export const modelVersion = '1';
+export const explanation = {};
+
+export const INPUT_DETAILED = 'detailed_input';
+export const INPUT_NON_DETAILED = 'non_detailed_input';
+
+// car categories for non-detailed input (without brand name and model)
+// categorized by size and type
+const GENERIC_CATEGORIES = {
```

Comment:
Is this used anywhere?

Reply:
Thank you for reviewing:)
I think this should be moved to a .json file (separate from .json file with car models with brand name and year). This was an example of how a generic car model (without brandname and year) can be defined.

Reply:
Actually, rather than having separate .json file for generic model and detailed model, I could put them in one car_mode.json file, if that is better.